### ISSUE TITLE
MediatR - version update

### ISSUE THREAD

**Issue by dj-1979** [PacktPublishing/Microservices-Design-Patterns-in-.NET#3]
hi @trevoirwilliams , i am not sure whether i am on the right branch. The new version of mediatr has got a different implementation in program.cs. Is it ok for me to make changes or you will make it?

**Comment by trevoirwilliams** [comment]
Indeed. You can submit the PR, and we will see. At the time of writing, this was the method.
